terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,64 @@
 provision_custom_search = False
 
 
+def content_security_policy() -> str:
```

Comment:
Please follow precedent by moving this utility function to where the other utility functions reside. WIthin those function, pick a suitable place. If you can't find one, or if in doubt, append, don't prepend.